main.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'app'
+
+def display_menu
+  puts 'Please choose an option by entering a number: '
+  CHOICES.each { |key, value| puts "#{key} - #{value.to_s.tr('_', ' ')}" }
+  gets.chomp.to_i
+end
+
+def handle_option(app, option)
+  choice = CHOICES[option]
+  if choice
+    app.send(choice)
+  else
+    puts 'Invalid option. Please choose a valid option'
+  end
+end
+
+def main
+  app = App.new
+
+  loop do
+    option = display_menu
+    handle_option(app, option)
+  end
+end
+
+main
```

Comment:
- [ ] Please always double-check your work before submitting the task for review, it seems like you submitted the project but maybe forgot to add the `CHOICES`, which does contain the list of `menus`. I tried to look for the choices but I didn't get them because it doesn't exist and because of it I am unable to check if your `menu` working. Kindly implement the functionality as per the `project` requirement to pass this milestone.

![image](https://github.com/peter-mwovi/school-library/assets/66476329/75fb6710-547b-41a4-9e66-a13274e1cfd0)